airbyte-integrations/bases/base-normalization/snowflake.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM fishtownanalytics/dbt:0.21.1
```

Comment:
are we creating a special dockerfile for snowflake because it's DBT yamls deviate from the standard yamls?

Reply:
yeah. Chris [suggested](https://docs.google.com/document/d/1DoXRj2kMvnO-4C1EIWLMxtaNgVJ38kSPQtz1XV-0kso/edit?disco=AAAATjThWHE) modifying `entrypoint.sh` to use the modified yaml instead of publishing a new docker image, but my thinking was to keep entrypoint.sh integration-agnostic.